### DISCUSSION TITLE
About knowledge

### DISCUSSION THREAD

**Discussion by Rockyread** [community/community#63191] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hi everyone,
Hello everyone!. Is it possible to communicate and learn from a mentor on the GitHub site? I would like to communicate/ exchange experience and knowledge on programming.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Rockyread! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Rockyread** [reply]
THANKS

**Comment by mehulrajdev** [comment, marked as answer]
Yes maybe

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
